data/json/recipes/recipe_electronics.json

```diff
@@ -2339,5 +2339,100 @@
     [ [ "amplifier", 1 ] ],
     [ [ "cable", 2 ] ]
   ]
+},{
+  "result" : "network_mod",
+  "type" : "recipe",
+  "category" : "CC_ELECTRONIC",
+  "subcategory" : "CSC_ELECTRONIC_PARTS",
+  "skill_used" : "electronics",
+  "difficulty" : 5,
+  "skills_required" : [ [ "electronics", 5 ], [ "computers", 4 ] ],
+  "time" : 30000,
+  "reversible" : true,
+  "autolearn" : true,
+  "qualities" : [ { "id" : "SCREW", "level" : 2 } ],
+  "tools": [
+    [
+      [ "soldering_iron", 5 ],
+      [ "toolset", 5 ]
+    ]
+  ],
+  "components": [
+    [ [ "antenna", 1 ] ],
+    [ [ "transponder", 1] ],
+    [ [ "receiver", 1 ] ],
+    [ [ "amplifier", 1 ] ],
+    [ [ "processor", 1 ] ],
+    [ [ "cable", 5 ] ],
+    [ [ "e_scrap", 2 ] ],
+    [ [ "scrap", 2 ] ]
+  ]
+},{
+  "result" : "notice_screen",
```

Comment:
Adding recipes that are used in only one place leads to bloating of the crafting screen. Would be better if you just included the requirements in the construction.

Reply:
Good point. Will change.